fvm/evm/evm_test.go

```diff
@@ -965,6 +990,29 @@ func TestEVMAddressDeposit(t *testing.T) {
 			expectedBalance := types.OneFlowBalance
 			bal := getEVMAccountBalance(t, ctx, vm, snapshot, addr)
 			require.Equal(t, expectedBalance, bal)
+
+			// block executed event, make sure TotalGasUsed is non-zero
+			blockEvent := output.Events[3]
+
+			assert.Equal(
+				t,
+				common.NewAddressLocation(
+					nil,
+					common.Address(sc.EVMContract.Address),
+					string(types.EventTypeBlockExecuted),
+				).ID(),
+				string(blockEvent.Type),
+			)
+
+			ev, err := ccf.Decode(nil, blockEvent.Payload)
+			require.NoError(t, err)
+			cadenceEvent, ok := ev.(cadence.Event)
+			require.True(t, ok)
+
+			blockEventPayload, err := types.DecodeBlockEventPayload(cadenceEvent)
+			require.NoError(t, err)
+			require.NotEmpty(t, blockEventPayload.Hash)
+			require.Greater(t, blockEventPayload.TotalGasUsed, uint64(0))
```

Comment:
Let's match it against an exact value

Reply:
Updated in https://github.com/onflow/flow-go/pull/6015/commits/2c69c3604c3c5dcb16071cfc9d297f878397c2be